### PR TITLE
Create stale.yml

### PR DESCRIPTION
**Pull request by lesv**
Automated close of stale issues / PR's

From: https://probot.github.io/apps/stale/

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
```
Comment (kurtisvg) [inline .github/stale.yml:3]: Is this _issues_ or _PRs_? (Or both?) 

I'm not sure if we should be closing issues, but I agree we should be closing PRs

Comment (lesv) [reply .github/stale.yml:3]: Both, I agree w/ you on issues, but expect it will force us to comment.

Comment (lesv) [reply .github/stale.yml:3]: I think I have a few changes to this.  Hold off for a bit.

Comment (lesv) [reply .github/stale.yml:3]: PTAL

**Comment by lesv** [comment]
PTAL

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```
Comment (kurtisvg) [inline .github/stale.yml:6]: What about DO NOT MERGE?

Comment (lesv) [reply .github/stale.yml:6]: We shouldn't have them as PR's, if they are being used by docs, they should be on official branches we review.  If a DNM isn't touched for 30 days, someone should ping.

**Pull request merged**